vale/styles/jargon/TechnologySpecials.yml

```diff
@@ -0,0 +1,40 @@
+---
+extends: substitution
+message: Use '%s' instead of '%s'
+level: warning
+ignorecase: true
+scope: raw
```

Comment:
I found a way to do not use raw, so I will be able to merge these rules with Technology.yml

This was the only reason to split into separate files

Reply:
I understand, but the reference is a problem. See my comment below.

Reply:
I find a way to move them back with the others @Jason3S helped me by suggesting using \b

it helped me to check negative behind rules